security/cmd/node_agent_k8s/main.go

```diff
@@ -80,7 +81,7 @@ func main() {
 type mockCAClient struct {
 }
 
-func (c *mockCAClient) CSRSign(csrPEM []byte, /*PEM-encoded certificate request*/
+func (c *mockCAClient) CSRSign(ctx context.Context, csrPEM []byte, /*PEM-encoded certificate request*/
```

Comment:
Why we need this "/*PEM-encoded certificate request*/", i don't think it's a proper comment for subjectID.

Reply:
and I think csrPEM itself is clear enough, btw :)

Reply:
removed comment for this param(here and other places).